l2geth/core/vm/evm.go

```diff
@@ -120,7 +120,25 @@ func run(evm *EVM, contract *Contract, input []byte, readOnly bool) ([]byte, err
 		// OVM_ENABLED
 		// Only log for non `eth_call`s
 		if evm.Context.EthCallSender == nil {
-			log.Debug("Calling contract", "ID", evm.Id, "Address", contract.Address().Hex(), "Data", hexutil.Encode(input))
+			var isUnknown = true
```

Comment:
reminder to remove this temp commit before we ship

Reply:
done, thank you.